astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2256,6 +2286,25 @@ def __init__(
         for local_name, node in self.implicit_locals():
             self.add_local_node(node, local_name)
 
+    @property
+    def doc(self) -> Optional[str]:
+        """The class' docstring."""
+        warnings.warn(
+            "The 'ClassDef.doc' attribute is deprecated, "
+            "use 'ClassDef.doc_node.value' instead.",
+            DeprecationWarning,
+        )
+        return self._doc
+
+    @doc.setter
+    def doc(self, value: Optional[str]) -> None:
+        warnings.warn(
+            "The 'ClassDef.doc' attribute is deprecated, "
+            "use 'ClassDef.doc_node.value' instead.",
```

Comment:
```suggestion
            "Setting the 'ClassDef.doc' attribute is deprecated, "
            "use 'ClassDef.doc_node.value' instead.",
```

Reply:
I intended this PR to only deprecate `accessing` and doing `setting` in a later PR. See (future) other comment.